frontend/models/product-list/types.ts

```diff
@@ -1,6 +1,7 @@
 export interface ProductSearchHit {
    objectID: string;
    title: string;
```

Comment:
Do we expect to still use `title` in some places?

Reply:
I'm not sure.. But you're right, I could have gotten rid of it for the time being.. If I'll do some clean up I'll get rid of it